specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/Databases.json

```diff
@@ -30,6 +30,12 @@
           {
             "$ref": "#/parameters/ServerNameParameter"
           },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
           {
```

Comment:
can this parameter be added in the end ?

Reply:
in the code, this parameter is last but the autogeneration put it here. We don't want to move it because this will cause conflicts when updating in the future.